src/site/_data/i18n/browser_compat.yml

```diff
@@ -1,11 +1,35 @@
 browser_support:
   en: Browser support
```

Comment:
@samthor 
What's the context of this text?

Reply:
I think this is @devnook 's "compatibility widget".

I suspect your (reasonable) concern is that the translations may have been done without any serious context on each string. If it's not high-quality we may need to find a way to feed context to translators, perhaps through a "_description" key here (i.e., adjacent to but not a language code).

Reply:
yeah, translation here looks correct and direct, but it may be wrong in different context. In that context, I'd suggest 
"지원되는 브라우저" and I guess this translation is more natural in most context.

Reply:
> I think this is @devnook 's "compatibility widget".
> 
> I suspect your (reasonable) concern is that the translations may have been done without any serious context on each string. If it's not high-quality we may need to find a way to feed context to translators, perhaps through a "_description" key here (i.e., adjacent to but not a language code).

@samthor While @devnook helped a lot with providing additional context for our translators, the main issue here is that with this particular case it's often impossible to run Linguistic QA on a localized build, as these strings seem to be rendered only for the pages/articles that have been translated (while many pages are in English only).  Of course, going forward, any additional description/context would help.